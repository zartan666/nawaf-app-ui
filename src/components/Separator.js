import React from 'react';
import {StyleSheet, View} from 'react-native';
import colors from './../common/colors';

const Separator = ({style}) => <View style={[styles.container, style]} />;

Separator.propTyes = {
  style: View.propTypes.style,
};

const styles = StyleSheet.create({
  container: {
    height: 0.5,
    backgroundColor: colors.smokeGreyLight,
  },
});

export default Separator;
