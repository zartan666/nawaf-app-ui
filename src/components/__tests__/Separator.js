import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Separator from './../Separator';

const navigator = {
  pop: jest.fn(),
};

test('renders null', () => {
  const tree = renderer.create(<Separator />).toJSON();
  expect(tree).toMatchSnapshot();
});
