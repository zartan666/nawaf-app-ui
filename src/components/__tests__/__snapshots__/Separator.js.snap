// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders null 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#e3e7ed",
        "height": 0.5,
      },
      undefined,
    ]
  }
/>
`;
